Form/Type/LegacyLegacySettingType.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Craue\ConfigBundle\Form\Type;
+
+use Symfony\Component\OptionsResolver\OptionsResolverInterface;
+
+/**
+ * for Symfony < 2.7
+ *
+ * @author Christian Raue <christian.raue@gmail.com>
+ * @copyright 2011-2015 Christian Raue
+ * @license http://opensource.org/licenses/mit-license.php MIT License
+ */
+class LegacyLegacySettingType extends LegacySettingType {
+
+	/**
+	 * {@inheritDoc}
+	 */
+	public function setDefaultOptions(OptionsResolverInterface $resolver) {
```

Comment:
Done in #23. I remember having tried some simplifications in extending those types, but got notices anyway. Trying again now seems to work fine.